eva-uow/src/main/kotlin/com/razz/eva/uow/Persisting.kt

```diff
@@ -23,8 +24,18 @@ import java.util.UUID.randomUUID
 class Persisting(
     private val transactionManager: TransactionManager<*>,
     private val modelRepos: ModelRepos,
-    private val eventRepository: EventRepository
+    private val eventRepository: EventRepository,
+    private val eventPublisher: EventPublisher
```

Comment:
3 options were considered: 
- add in `Persisting` as parameter as implemented here. Drawbacks: `Persisting` dependency on `EventPublisher` which is not used in most cases ;
- make `Persisting` interface and implement `InproccessEventPublishingPersisting::persist`. Drawbacks: OOP bloat;
- do publishing in `EventRepository::add`. Drawbacks: publishing will be done in transaction and could affect db performance;

Reply:
Current implementation is okey @kgribov @spospekhov approve